modules/core_api/fsw/inc/cfe_es_extern_typedefs.h

```diff
@@ -0,0 +1,573 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/**
+ * @file
+ *
+ * Declarations and prototypes for cfe_es_extern_typedefs module
+ */
+
+#ifndef CFE_ES_EXTERN_TYPEDEFS_H
+#define CFE_ES_EXTERN_TYPEDEFS_H
+
+/* This header may be generated from an EDS file,
+ * tools are available and the feature is enabled */
+#ifdef CFE_EDS_ENABLED_BUILD
+
+/* Use the EDS generated version of these types */
+#include "cfe_es_eds_typedefs.h"
+
+#else
+/* Use the local definitions of these types */
+
+#include "common_types.h"
+#include "cfe_resourceid_typedef.h"
+#include "cfe_mission_cfg.h"
+
+/**
+ * @brief Label definitions associated with CFE_ES_LogMode_Enum_t
+ */
+enum CFE_ES_LogMode
```

Comment:
For this and the other enums, why specify all of the values?  utilize the compiler/language which auto increments... use prescribed values if you are purposely skipping or starting someplace.

Reply:
Even though they are enums, there are ground dependencies on these specific values.